### ISSUE TITLE
Discussion about QM refactor ideas

### ISSUE THREAD

**Issue by HoofedEar** [space-wizards/space-station-14#7592]
Quartermaster is in a very interesting spot in terms of job roles. They are technically a departmental head (with a cape, their own office, and a department remote). However, they are the only department head that does not have command access and can be played as an antag. It feels very weird to have a pseudo-head role that can become antag. 

If we look to other codebases for guidance, we seem to be mimicking Fulpstation and Paradise (probably others) where the QM does oversee Cargo, but they are not a Command Head. Meanwhile, on /tg/ and Goon, their Quartermasters = Cargo Techs, with HoP and CE being their department heads respectively. 

I have seen a few different ideas thrown around about how to handle QM.
1) Give QM command access and remove antag, making them a proper department head.
2) Remove QM entirely because their role is a bit redundant. HoP will then become the department head of Cargo.
3) Simply just remove antag from QM.

Personally, I’m cool with option 1 and 3, since it’s less work for mappers 

Discuss!

**Comment by Emisse** [comment]
I'd honestly lean more towards 1. HoP is already so busy overseeing all of supply is a lot.

**Comment by TimrodDX** [comment]
The QM shouldn't be a head, and nothing should be changed there. There are two key reasons for this.

The first is a balance issue from the perspective of an antagonist. The game is balanced around the idea that everyone has to leave their departments and highly-secure offices: either to find food or drink to prevent a death from starvation, or to do some key aspect of their job. If the HoP no longer has to oversee Cargo, they now have one fewer reason to ever leave their r-walled compound.

The second is a balance issue from the perspective of division of responsibilities. The two dangerous things the HoP oversees are Cargo and their own ID card. If the HoP no longer has to oversee Cargo, that effectively reduces them to a living robot whose job is to hand out ID changes. There have already been calls to remove the HoP entirely, and I think reducing the HoP's actual responsibilities any further would just make the problem worse.

**Comment by HoofedEar** [comment]
> The first is a balance issue from the perspective of an antagonist. The game is balanced around the idea that everyone has to leave their departments and highly-secure offices: either to find food or drink to prevent a death from starvation, or to do some key aspect of their job. If the HoP no longer has to oversee Cargo, they now have one fewer reason to ever leave their r-walled compound.

> The second is a balance issue from the perspective of division of responsibilities. The two dangerous things the HoP oversees are Cargo and their own ID card. If the HoP no longer has to oversee Cargo, that effectively reduces them to a living robot whose job is to hand out ID changes. There have already been calls to remove the HoP entirely, and I think reducing the HoP's actual responsibilities any further would just make the problem worse.

You make a good point about HoP needing a reason to leave for balance reasons, but wouldn't this be an argument to _remove_ QM? In my experience, I hardly ever see HoP stop by Cargo. Removing QM would make HoP need to check on Cargo more often.

**Comment by Fishfish458** [comment]
> You make a good point about HoP needing a reason to leave for balance reasons, but wouldn't this be an argument to _remove_ QM? In my experience, I hardly ever see HoP stop by Cargo. Removing QM would make HoP need to check on Cargo more often.

QM is a necessary go between IMO, as a department cargo is both incoming salvage and deliveries. They're the ones that should be accepting, denying the orders on a case by case basis and managing cargo points spent which just wouldn't be feasible with the amount of requests that HoP is usually asked to do at their own office.

**Comment by HoofedEar** [comment]
> QM is a necessary go between IMO, as a department cargo is both incoming salvage and deliveries. They're the ones that should be accepting, denying the orders on a case by case basis and managing cargo points spent which just wouldn't be feasible with the amount of requests that HoP is usually asked to do at their own office.

I agree, so maybe a good compromise would be to remove antag from QM? Since they are sudo-head of Cargo, with power over that department, I don't think they should be able to spawn antag

**Comment by Fishfish458** [comment]
> > QM is a necessary go between IMO, as a department cargo is both incoming salvage and deliveries. They're the ones that should be accepting, denying the orders on a case by case basis and managing cargo points spent which just wouldn't be feasible with the amount of requests that HoP is usually asked to do at their own office.
> 
> I agree, so maybe a good compromise would be to remove antag from QM? Since they are sudo-head of Cargo, with power over that department, I don't think they should be able to spawn antag

I'm definitely pretty fine with that myself, usually an antag QM leads to more cases of cargonia.

**Comment by Peptide90** [comment]
50/50 on removing QM antag. They're kind of in an interesting spot in that they're probably the most corruptible staff member for taking bribes and smuggling. It is a powerful role but I've remained adamant they should be a supervisor and not a head. Then again making them head of salvage and cargo could be neat and end the argument once and for all about them getting an office, I just think it'll be an OP low hanging fruit target to get a command headset and access card since they're low priority to protect.

When we have cargo shuttles etc the QM will be even more important for inspecting cargo and managing that whole procedure that shouldn't be left to possible (multiple) antag cargo techs.

**Comment by retequizzle** [comment]
I'm all for removing QM as an antag role, but I'm equally for removing supply boards from the QM's locker and moving it to HoP's locker instead. Even as things are right now, QM is arguably more easy-mode as antag than any other role because you can take your supply board, grab a salvage suit and literally never be seen again with one of your greentexts easily done.  At that point, turn off suit sensors, and walk around like a late-join salvage tech until you can do your other objectives. 

I'm also all for either making QM a full department head with command access, or removing QM as a special designation at all. The idea that HoP needs to remain the head in order to force them out of their office more is kind of flawed, in that HoP already rarely leaves their office and in some cases will powergame moving drinks and prepackaged food to their office to reduce any need for leaving. In addition, I can count the times I've spoken to the HoP in cargo while playing as QM on one hand, even if it's just them stopping by to ask how things are going. HoP being the head of cargo proper doesn't really work when they never come to cargo. I am not sure if it's a map issue, a role issue, or a skill issue in that regard though. 

tl;dr - yes remove QM antag role it's a big dum powergame and give them proper command access so they have to act the part, or remove QM as special designation and reinforce HoP job specs to include supply as part of their rotation, up to and including having a spare supply board in their locker
